src/AvtaleSide/steg/BeregningTilskudd/BeregningTilskuddSteg.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Innholdsboks from '@/komponenter/Innholdsboks/Innholdsboks';
+import { Normaltekst, Systemtittel, Undertittel } from 'nav-frontend-typografi';
+import { Context, medContext } from '@/AvtaleContext';
+import RadioPanelGruppeHorisontal from '@/komponenter/form/RadioPanelGruppeHorisontal';
+import { Column, Container, Row } from 'nav-frontend-grid';
+import ValutaInput from '@/komponenter/form/ValutaInput';
+import ProsentInput from '@/komponenter/form/ProsentInput';
+import SelectInput from '@/komponenter/form/SelectInput';
+import VerticalSpacer from '@/komponenter/layout/VerticalSpacer';
+import LagreKnapp from '@/komponenter/LagreKnapp/LagreKnapp';
+import KontonummerInput from '@/komponenter/form/KontonummerInput';
+
+const feriepengeAlternativer = (erOver60: boolean) => {
+    const satser = erOver60 ? [0.12, 0.143] : [0.102, 0.125];
+    return satser.map((sats: number) => ({
+        label: (sats * 100).toFixed(1) + ' %',
+        value: sats.toString(),
+    }));
+};
+const arbeidsgiveravgiftAlternativer = () => {
+    const satser = [0.141, 0.106, 0.064, 0.051, 0.079];
+    return satser.map((sats: number) => ({
+        label: (sats * 100).toFixed(1) + ' %',
+        value: sats.toString(),
+    }));
+};
+const BeregningTilskuddSteg = (props: Context) => {
+    const { settAvtaleVerdi, avtale } = props;
```

Comment:
Føler det kan bli vanskeligere å se forskjell på hva som kommer fra props/context og ikke når man extracter sånn, men ser det er greit å slippe å skrive `.props` mange ganger også. 🤷‍♂️

Reply:
Jeg tror Object Destructuring er en bra ting, så jeg gjør det for å forbedre lesbarhet.